### PR TITLE
docs: added LVM and ZFS FAQs, to the respective sections in the local storage user guide.

### PR DESCRIPTION
**Pull request by datacore-gthomas**

### CODE REVIEW

**Comment by datacore-gthomas** [comment]
@avishnu @Abhinandan-Purkait please review the FAQ section and approve/suggest changes, if required.

**Comment by datacore-gthomas** [comment]
@vishnuitta @Abhinandan-Purkait, please review/share your feedback if any. If the changes are good to go, please approve/merge the PR.
The FAQ content for ZFS and LVM were moved into the General FAQ section and the cross-referencing and hyperlinks links were corrected.

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:462]: Engine, right? as it's a.k.a?

File: docs/main/concepts/data-engines/data-engines.md
Comment (avishnu) [inline docs/main/concepts/data-engines/data-engines.md:132]: This change was made in a different PR, you may need to rebase.

File: docs/main/concepts/architecture.md
Comment (avishnu) [inline docs/main/concepts/architecture.md:169]: Wasn't this made in an earlier PR? You need to rebase

File: docs/main/concepts/data-engines/local-storage.md
Comment (avishnu) [inline docs/main/concepts/data-engines/local-storage.md:101]: Why is ../main added here? This can break once we create branch folders. Please re-check.

File: docs/main/concepts/data-engines/replicated-storage.md
Comment (avishnu) [inline docs/main/concepts/data-engines/replicated-storage.md:33]: This seems not relevant here. It is mentioned local storage when the doc is about replicated storage?

File: docs/main/concepts/data-engines/replicated-storage.md
Comment (avishnu) [inline docs/main/concepts/data-engines/replicated-storage.md:49]: Same comment about ../main as above

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:25]: The path should be relative and not include main

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:528]: Maintainers of Mayastor (Replicated Storage) have performed benchmarking tests in collaboration with Intel...

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:584]: Yes, snapshots and clones support is presently limited to volumes with only one replica. Snapshot and clone support for multi-replica volumes is planned in the upcoming release.

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:594]: as the current version of SPDK requires ARMv8 Crypto extensions...

@tiagolobocastro can you confirm if this limitation needs to be updated?

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:618]: remove extra blank spaces in 'Replicated Storage   has introduced...'

File: docs/main/introduction-to-openebs/introduction-to-openebs.md
Comment (avishnu) [inline docs/main/introduction-to-openebs/introduction-to-openebs.md:81]: ../main should not be present.

File: docs/main/quickstart-guide/deploy-a-test-application.md
Comment (avishnu) [inline docs/main/quickstart-guide/deploy-a-test-application.md:226]: avoid ../main in the path.

File: docs/main/quickstart-guide/deploy-a-test-application.md
Comment (avishnu) [inline docs/main/quickstart-guide/deploy-a-test-application.md:227]: avoid ../main in the path.

File: docs/main/quickstart-guide/installation.md
Comment (avishnu) [inline docs/main/quickstart-guide/installation.md:189]: avoid ../main in the path

File: docs/main/quickstart-guide/installation.md
Comment (avishnu) [inline docs/main/quickstart-guide/installation.md:193]: avoid ../main in the above paths

File: docs/main/user-guides/replicated-storage-user-guide/rs-installation.md
Comment (avishnu) [inline docs/main/user-guides/replicated-storage-user-guide/rs-installation.md:282]: Here, it should be engine, again this must have been a part of an earlier PR, you may need to rebase

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md:1]: Are the below contents repeated in the master faq?

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/faq.md
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-zfs/faq.md:1]: Are the below contents repeated in the master faq?

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md
Comment (datacore-gthomas) [reply docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md:1]: No

File: docs/main/user-guides/local-storage-user-guide/local-pv-zfs/faq.md
Comment (datacore-gthomas) [reply docs/main/user-guides/local-storage-user-guide/local-pv-zfs/faq.md:1]: no

File: docs/main/concepts/data-engines/data-engines.md
Comment (datacore-gthomas) [reply docs/main/concepts/data-engines/data-engines.md:132]: This change was not there, Vishnu. I checked.

File: docs/main/user-guides/replicated-storage-user-guide/rs-installation.md
Comment (datacore-gthomas) [reply docs/main/user-guides/replicated-storage-user-guide/rs-installation.md:282]: Corrected, This change was not there, Vishnu. I checked.

File: docs/main/concepts/architecture.md
Comment (datacore-gthomas) [reply docs/main/concepts/architecture.md:169]: This change was not there, Vishnu. I checked.

File: docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md
Comment (avishnu) [reply docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md:1]: Hmm, not sot sure, I think there is a repetition. I see a question like "How to add custom topology key in the Local PV LVM driver?" in the master faq as well. Please check once again.

Comment (datacore-gthomas) [reply docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md]: I created a separate FAQ page for LVM and ZFS in their respective folders initially, later I moved all the FAQs into the master FAQ. So there is only one now. No repetition.

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:528]: Should this be "has been done" instead of "was done"? Just want to know your thoughts.

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:594]: which are not currently available for Pi?

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:618]: Extra space in "kubectl delete PV "

File: docs/main/faqs/faqs.md
Comment (datacore-gthomas) [reply docs/main/faqs/faqs.md:528]: Was done - seems alright since the statement is in past tense.

File: docs/main/faqs/faqs.md
Comment (datacore-gthomas) [reply docs/main/faqs/faqs.md:594]: This is alright, as the "are" refers to "ARMv8 Crypto extensions" which is in the plural form.

File: docs/main/faqs/faqs.md
Comment (datacore-gthomas) [reply docs/main/faqs/faqs.md:618]: The extra spacing is not showing up in the VS Code. Nothing to remove!

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:183]: @balaharish7 @datacore-gthomas this is the old way of installing. We need to update this part or provide a link to the main installation.

File: docs/main/faqs/faqs.md
Comment (avishnu) [reply docs/main/faqs/faqs.md:594]: I meant to combine the two sentences as follows:
Replicated Storage does not run on Raspbery Pi as the current version of SPDK requires ARMv8 Crypto extensions which are not currently available for Pi.

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:604]: Why do Replicated Storage's IO engine pods...

File: docs/main/faqs/faqs.md
Comment (avishnu) [inline docs/main/faqs/faqs.md:606]: "heart of the Replicated Storage's IO engine pod. When the IO engine pod is deployed on a cluster, it is..."

"CPU resource limits for the IO engine daemonset"

File: docs/main/faqs/faqs.md
Comment (datacore-gthomas) [reply docs/main/faqs/faqs.md:183]: removed this FAQ.

File: docs/main/faqs/faqs.md
Comment (datacore-gthomas) [reply docs/main/faqs/faqs.md:594]: got it. done.

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**
